### ISSUE TITLE
Probabilistic repetitions for Storm and Birds typically

### ISSUE THREAD

**Issue by BrunoVernay** [rafaelmardojai/blanket#81]
Some sounds could benefits from being NOT continuously repeated. 
Birds and Storm are the typical example.  It could even be improved by steps:

1. the current sound could be played randomly, not looped.
2. The probability to have the sound played over a period of time could be configured with a second slider (not frequent  to  very frequent)
3. The "Bird Sound" could be composed from multiple bird sound playing randomly with more or less probability
4. ...

**Comment by ArjunSingh99** [comment]
I agree. The thunderstorm sound sounds very artificial with the way it's being looped. I think randomizing some of these sounds would make a world of difference. Having a probability slider sounds like complicating it a little bit, maybe that could be included in an "Advanced" tab.

**Comment by n8willis** [comment]
For the record, probabilistically varying sounds is precisely what Boodler is for: https://github.com/erkyrath/boodler (or a somewhat-newer fork, https://github.com/beaugunderson/boodler-redux).

There is also a decent library of existing soundscapes for Boodler out there; it likely is not well known because it was not Linux-first, but this and a few of the other open issues relate directly to the problem it solves.

**Comment by q2dg** [comment]
For me, solving this issue should be the priority number 1. Meanwhile, these sounds are a no-go
